packages/site/src/pages/Article.tsx

```diff
@@ -5,7 +5,7 @@ import { ErrorProjectNotFound } from './ErrorProjectNotFound';
 import type { PageLoader } from '../types';
 
 export function ArticlePage({ article }: { article: PageLoader }) {
-  const { hide_title_block, hide_footer_links } = article.frontmatter?.design ?? {};
+  const { hide_title_block, hide_footer_links } = (article.frontmatter as any)?.design ?? {};
```

Comment:
Some of this page-level customization may be a little kluge-y until we get site template option overrides in page frontmatter...

Reply:
Yep, can make it better as we go.